tests/fail/warnings_and_errors.py

```diff
@@ -2,4 +2,6 @@
 
 np.AxisError(1.0)  # E: Argument 1 to "AxisError" has incompatible type
 np.AxisError(1, ndim=2.0)  # E: Argument "ndim" to "AxisError" has incompatible type
-np.AxisError(2, msg_prefix=404)  # E: Argument "msg_prefix" to "AxisError" has incompatible type
+np.AxisError(
+    2, msg_prefix=404  # E: Argument "msg_prefix" to "AxisError" has incompatible type
```

Comment:
This is an interesting one where you do have to modify the output black gives you a bit because the exact line number matters. I don't think this should be a huge inconvenience going forward though.